public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -427,6 +427,12 @@ function serviceMapQuery(request: DataQueryRequest<TempoQuery>, datasourceUid: s
       }
 
       const { nodes, edges } = mapPromMetricsToServiceMap(responses, request.range);
+
+      // No handling of multiple targets assume just one. NodeGraph does not support it anyway, but still should be
+      // fixed at some point.
+      nodes.refId = request.targets[0].refId;
```

Comment:
This is neede for the dashboard flow to figure out which panel to use so now it correctly creates the dashboard with node graph panel.